### PR TITLE
provider/azurerm: Add example of Spark and Cassrandra on CentOS

### PR DESCRIPTION
**Pull request by anniehedgpeth**
[Build Status](https://travis-ci.org/harijayms/terraform)

This example provisions and deploys the following resources into an Azure subscription:

- Resource Group
- 3 NSGs (Master, Slave/Agent, Cassandra)
- 1 VNET
- 3 Subnets (Master, Slave/Agent, Cassandra)
- 3 PIPs (Master, Slave/Agent, Cassandra)
- 3 NICs (Master, Slave/Agent, Cassandra)
- 1 Availability Set
- 3 Storage Accounts (Master, Slave/Agent, Cassandra)
- 3 Storage Containers (Master, Slave/Agent, Cassandra)
- 1 Master VM
- Slave/Agent VMs based on count variable
- 1 Cassandra VM

The preferred method of running the scripts is the VM Extension, but there is currently an issue submitted for this that is preventing me from using that resource. [https://github.com/Azure/custom-script-extension-linux/issues/95](url)

**CI process:** 
```
terraform get, validate, plan to output, apply output, show
validate with azure CLI
terraform destroy
```

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -41,4 +41,4 @@ notifications:
 matrix:
   fast_finish: true
   allow_failures:
-  - go: tip
```
Comment (stack72) [inline .travis.yml]: please can you revert the line endings change on this file?

Comment (anniehedgpeth) [inline .travis.yml]: I believe this is changed back to original, @stack72 - let me know if it's still different

**Comment by anniehedgpeth** [comment]
@stack72 I added the return line on 45 of the .travis.yml and removed the crazy graph. Let me know how it looks!

**Comment by stack72** [comment]
Hi @anniehedgpeth 

Thanks for update this - this LGTM

Paul

**Pull request merged**